### PR TITLE
Set up Helmet to lock down security

### PR DESCRIPTION
**Pull request by jayzes**
https://www.npmjs.com/package/helmet

Confusing that there are now two `helmet`s in the project but whatever.

By default, this will disable DNS prefetch, secure frames, hide the
powered by header, enforce HSTS, set X-Download-Options for IE, prevent
MIME type sniffing, and add XSS protections. We can add in CSP and HPKP
later if we want.

### CODE REVIEW

**Comment by rynbyjn** [comment]
So the 3 vulnerabilities appear to be peer dependencies in the main dependencies of our project. Not quite sure how to resolve that. UglifyJS is part of `webpack` and apparently the handlebars warning is in something related to `postcss-url`.

File: package.json [hunk 1]
```diff
@@ -101,7 +101,9 @@
       "react/forbid-prop-types": [
         1,
         {
-          "forbid": ["any"]
+          "forbid": [
+            "any"
+          ]
```
Comment (rynbyjn) [inline package.json]: I think this is in the profile branch too, but hopefully shouldn't conflict since it was `npm install` that updated it.

**Comment by jayzes** [comment]
Sweet - I'll merge it after we launch watch a post, want to do a little testing to make sure this doesn't break anything.

**Comment by rynbyjn** [comment]
@jayzes is this still something we want to get into the webapp?

**Comment by jayzes** [comment]
Looks good on stage2 @rynbyjn - I tidied it up and fixed the merge conflicts, should be good to go.

**Pull request merged**